src/coreclr/jit/gentree.h

```diff
@@ -4287,6 +4300,10 @@ class CallArg
     unsigned m_tmpNum;
     // The type of the argument in the signature.
     var_types m_signatureType : 5;
+#ifdef TARGET_WASM
+    // The CorInfoType of argument in the signature.
+    CorInfoType m_signatureCorInfoType : 8;
```

Comment:
The field should also be copied here:

https://github.com/dotnet/runtimelab/blob/55680b0e768fd6ad4a23c12b8859266b79faedc9/src/coreclr/jit/gentree.cpp#L9133-L9146